### DISCUSSION TITLE
A vision system for classifying polymer details from the injection modeling process

### DISCUSSION THREAD

**Discussion by Wiktor010** [mrdbourke/pytorch-deep-learning#647] [General]
Hello, I would like to complete a project involving the creation of a vision system for classifying polymer details from the injection process. At the very beginning I had to write a script for the camera to process images. I have pre-processed these images. Now the images are fed to the input of CNN (SqueezeNet), which will classify a given detail into one of the defined classes. I am using a network model that is trained. I have a question for you if I have a model trained on ImageNet data. So is my task now just fine tuning and what next steps should I take? At the very end I would like to upload it to Raspberry Pi. classification should be at least 90%, and the processing time for one image should be less than 1500 ms.

**Comment by shisirkha** [comment]
Oh a nice beefy computer vision problem you can be a photographer to take photos of your data like 1000 samples of data in photo and use OpenCV python to do Realtime image recognition 

good luck with the project.
